sites/svelte.dev/src/routes/faq/+page.svelte

```diff
@@ -36,7 +32,7 @@
 		margin: 0 auto;
 		tab-size: 2;
 	}
-
+	/* 
```

Comment:
They are commented out but not deleted. Any reason for that? Is this something to discuss if this should change or not?

Reply:
Good catch!. They can be deleted, I will do that. Forgot some things during the testing phase

Reply:
Deleted